app/models/asset_field_type.rb

```diff
@@ -30,6 +30,7 @@ def data
       'asset': {
         'file_name': asset_file_name,
         'url': asset.url,
+        'image': !!image?,
```

Comment:
yes, but this method returns `nil` or `0`, which is why I made the other predicate method `is_image?`.